pkg/linear-pools/contracts/tetu-linear-pool/TetuShareValueHelper.sol

```diff
@@ -70,8 +70,8 @@ contract TetuShareValueHelper {
 
     function _getStrategyInvestedUnderlyingBalance(address wrappedTokenAddress) private view returns (uint256) {
         address tetuStrategy = _getTetuStrategy(wrappedTokenAddress);
-        if (address(tetuStrategy) == address(0)) {
-            return 0;
+        if (tetuStrategy == address(0)) {
```

Comment:
About this, in production I don't think tetuStrategy address will be 0. Still, we don't control the value of stragegy() function, so, if the cost of the "if-statement" is not expensive, maybe we should keep it? I left it, but changed the return value to _defaultRate (which is FixedPoint.ONE). @rabmarut, If you think it's safe, I can remove it completely

Reply:
Actually, I was just wondering if this is the safest course of action. I think there are four possible paths:

1. Don't worry about `strategy()` returning `address(0)`.
2. If `strategy()` returns `address(0)`, then revert the transaction.
3. If `strategy()` returns `address(0)`, then return a rate of 0.
4. If `strategy()` returns `address(0)`, then return a rate of `FixedPoint.ONE`.

After investigating Tetu's `SmartVault` code, I have learned the following:
- The strategy can be changed, so we cannot assume it is constant (that's fine; we don't assume this).
- The strategy can be `address(0)`, so we need to account for that case.
- The strategy cannot be changed to `address(0)` - it can only be `address(0)` if it has not yet been initialized.

So, with that conclusion, I think what you are doing now (`FixedPoint.ONE`) is correct, and we should add a comment explaining that this is the uninitialized case (same as total supply of 0).